prover/crates/bin/prover_fri/src/prover_job_processor.rs

```diff
@@ -186,7 +186,7 @@ impl Prover {
 #[async_trait]
 impl JobProcessor for Prover {
     type Job = ProverJob;
-    type JobId = u32;
+    type JobId = (L2ChainId, u32);
```

Comment:
Can it be `ChainAwareL1BatchNumber`?

Reply:
No, because it is not batch number, it is job id